nautobot/docs/user-guide/platform-functionality/objectmetadata.md

```diff
@@ -0,0 +1,129 @@
+# Object Metadata
+
+> metadata, noun: a set of data that describes and gives information about other data. -- Oxford Languages
+
+Nautobot's _Metadata_ feature set allows users and administrators to define data _about_ the network data (database records, or even individual database fields) recorded in Nautobot. This is an advanced feature set, targeting the needs of Enterprise Data Governance teams that desire the ability to track information about the data that Nautobot itself contains. Some examples of metadata that you might use in your organization:
+
+* Who is responsible for the accuracy of this data?
+* When (after what date) can this data be deleted?
+* What is the classification or risk level of this data?
+* Which system of record is this data originally sourced from?
+* When was this data last synced from that system into Nautobot? (Note that this is different than the `last_updated` field, as a sync may indicate "no change" since some previous sync date)
+
+While there are several data model extensibility features in Nautobot--like Custom Fields & Computed Fields--that aim to expand the existing network data model to suit your originization's specific needs, note that the Object Metadata feature set is not intended to represent data about the network. Rather, Object Metdata is a platform level feature that supports ancelary buisness process pertaining to the management of data within Nautobot. Still though, there are aspects that are likely to be useful to networking teams alike, for example, knowing who to talk to about correcting some device records, or identify the upstream system that is providing location data into Nautobot.
+
+This Object Metadata feature is entirely optional and very flexible in terms of the type of information it can support. At present, all data stored by Object Metdata is read-only in the web UI and the intention is that metdata is managed by Nautobot system intentions, namely SSoTs and REST API integrations. Thus, it is up to the maintainer of such integrations to implement support for tracking Object Metadata, but full ORM and REST API access is available.
```

Comment:
Not sure about "intentions"/"integrations" change

```suggestion
This Object Metadata feature is entirely optional and very flexible in terms of the type of information it can support. At present, all data stored by Object Metadata is read-only in the web UI and the intention is that metadata is managed by Nautobot system integrations, namely SSoTs and REST API integrations. Thus, it is up to the maintainer of such integrations to implement support for tracking Object Metadata, but full ORM and REST API access is available.
```